R/Task.R

```diff
@@ -70,6 +70,9 @@
 #'   Should sanity of data be checked initially at task creation?
 #'   You should have good reasons to turn this off (one might be speed).
 #'   Default is \code{TRUE}.
+#' @param spatial [\code{logical(1)}]\cr
+#'   Is the task spatial? I.e. does it contain coordinates ("x" and "y) which
+#'   shall be used for spatial partitioning?
```

Comment:
Can you expand on this a bit and explain how they are used for partitioning and how this partitioning is used please?